### ISSUE TITLE
Installation and run the game ?

### ISSUE THREAD

**Issue by zilveer** [ginuerzh/poker#4]
Hi,
How can I install the game and play it ?
Also is it fully finished ?

**Comment by MarlikAlmighty** [comment]
No, this is not a working draft, not structured crap. Although there are interesting thoughts. https://github.com/golang-standards/project-layout
